### PR TITLE
[BUG]: Revert code for treemap and histogram.

### PR DESCRIPTION
**Pull request by ruchika-narang**
### Description
Reverting code for histogram and treemap after the merge of antlr code for app crashes.

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by abasatwar — DISMISSED** [review]
Once the behavior is confirmed , we can merge this.

**Comment by ruchika-narang** [comment]
Hi @mengweieric ,

I am still working on this one, this will be ready for review by your late evening.

Thanks!

File: dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_config_panel_item.tsx [hunk 1]
```diff
@@ -113,60 +121,78 @@ export const DataConfigPanelItem = ({ fieldOptionList, visualizations }: any) =>
...
+          vizId: curVisId,
           data: {
+            ...userConfigs,
             dataConfig: {
-              metrics: updatedConfigList.metrics,
-              dimensions: updatedConfigList.dimensions,
-              breakdowns: updatedConfigList.breakdowns,
-              span: updatedConfigList.span,
+              ...userConfigs.dataConfig,
+              [GROUPBY]: configList[GROUPBY],
```
Comment (abasatwar) [inline dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_config_panel_item.tsx:140]: Try to use same format at all places.

**Review by mengweieric — APPROVED** [review]

**Review by kavithacm — APPROVED** [review]

File: dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_config_panel_item.tsx (hunk 1 above)
Comment (mengweieric) [reply dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_config_panel_item.tsx:143]: Hi just want to add one more support for @abasatwar's comment, please make replacements similar here for ALL visualizations not just this PR thanks!

Comment (ruchika-narang) [reply dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_config_panel_item.tsx:143]: Hi @mengweieric ,

We have made change in this PR itself for all instances of dimensions and metrics. Once this PR is merged all the other PRs will be rebased with main to get this PRs code. Please let me know if this works. Thanks!

**Review by mengweieric — APPROVED** [review]

**Review by sejli — APPROVED** [review]

**Pull request merged**